protocol/avdl/keybase1/teams.avdl

```diff
@@ -592,7 +594,20 @@ protocol teams {
   // Includes admins of the specified team only if they are also admins of ancestor teams.
   array<TeamMemberDetails> teamImplicitAdmins(int sessionID, string teamName);
 
-  AnnotatedTeamList teamList(int sessionID, string userAssertion, boolean all, boolean includeImplicitTeams);
+  // List known team memberships for given userAssertion or current
+  // user if userAssertion is not given. This RPC uses fast server-
+  // trust path and returned list is not checked.
+  AnnotatedTeamList teamListUnverified(int sessionID, string userAssertion, boolean includeImplicitTeams);
```

Comment:
@mlsteele is this fine now? thanks and sorry for going back and forth so much on this

Reply:
Yes I like this, feels safe. Thanks!